src/chocolatey/infrastructure.app/services/TemplateService.cs

```diff
@@ -188,5 +187,21 @@ public void generate_file_from_template(ChocolateyConfiguration configuration, T
             _fileSystem.create_directory_if_not_exists(_fileSystem.get_directory_name(fileLocation));
             _fileSystem.write_file(fileLocation, template, encoding);
         }
+
+        private string parse_template_path(ChocolateyConfiguration configuration)
+        {
+            if (!configuration.NewCommand.TemplateName.Equals("default"))
+            {
+                var rawRelativePath = _fileSystem.combine_paths(_fileSystem.get_current_directory(), configuration.NewCommand.TemplateName);

```

Comment:
I don't think this is required, you should be able to call `_filesystem.get_full_path()` directly with the `TemplateName`. That would allow absolute paths to work as well as relative paths.